File: src/main/java/ast/ASTReturnStmnt.java
package ast;

public class ASTReturnStmnt extends ASTNode {
    public ASTExpr val;

    public ASTReturnStmnt(ASTFileLocation loc, ASTExpr val){
        super(loc);
        this.val = val;
    }

    public void print(int i){
        printIndent(i);
        System.out.println("ASTReturnStmnt:");
        val.print(i+1);
    }
}

File: src/main/kotlin/ast/ASTReturnStmnt.kt
package ast

class ASTReturnStmnt(loc: ASTFileLocation, val value: ASTExpr?) : ASTNode(loc) {

    override fun print(i: Int) {
        printIndent(i)
        println("ASTReturnStmnt:")
        value?.print(i + 1)
    }
}